### PR TITLE
Rollback feature (for test)

### PR DESCRIPTION
**Pull request by elmertoft**
Re #44

### CODE REVIEW

File: Bender/services/Github.js [hunk 1]
```diff
@@ -3,6 +3,27 @@ let Promise = require('bluebird')
 
 const githubAPIRepoURL = 'https://api.github.com/repos/connectedcars/connectedcars'
 
+function getLatestMasterCommitHash() {
+  return new Promise((resolve, reject) => {
+        axios.get(`${githubAPIRepoURL}/branches/master`, {
```
Comment (kevinsimper) [inline Bender/services/Github.js]: axios is a promise, should you don't need to create a new promise :)

Comment (elmertoft) [inline Bender/services/Github.js]: @kevinsimper I've changed it now :)

File: Bender/services/Github.js [hunk 2]
```diff
@@ -3,6 +3,25 @@ let Promise = require('bluebird')
...
+      'Authorization': `token ${process.env.GITHUB_TOKEN}`,
+      // The Protected Branch API is currently available for developers to preview. During the preview period, the API may change without advance notice.
+      // See: https://developer.github.com/v3/repos/branches/#get-branch
+      // To access the API during the preview period, you must provide a custom media type in the Accept header:
+      'Accept': 'application/vnd.github.loki-preview+json'
+    }
+      }).then(resBranch => {
+        // Returns the eight first characters of the commit hash
+        return resBranch.data.commit.sha.substr(0,8);
+      }).catch(error => {
```
Comment (kevinsimper) [inline Bender/services/Github.js]: Don't handle the `catch` if you just throw a new rejection, then let the parent handle it

File: Bender/services/DeployToStaging.js [hunk 3]
```diff
@@ -1,27 +1,51 @@
...
-    deploymentTag = hash;
-    postMessageToSlack("Deploying hash: " + hash + " to staging...", "tech");
-    console.log("Deploying tag: " + hash + " to staging...");
-  }
-  else {
-    postMessageToSlack("Deploying to staging...", "tech");
-    console.log("Deploying to staging...");
-  }
+exports.deployToStaging = (commitHash, bot = '', botMessage = '') => {
+  let _reply = (replyMessage) => {
```
Comment (kevinsimper) [inline Bender/services/DeployToStaging.js]: I don't know how good idea it is too pull in the the bot object, it goes against making code that are simple and does one thing well. Why not create a function that returns something and then keep the bot logic where it exists?

**Pull request closed without merging**